server/users/tests/routes.test.ts

```diff
@@ -79,5 +79,62 @@ describe("Test users routes", () => {
     );
   });
 
+  // TODO: add content to the body of this test
   test("Correctly updates the cache after user pins board", async function () {});
```

Comment:
trick: you can use `test.todo` to signal that the test needs to be written. See: https://github.com/essential-randomness/bobaserver/blob/realm-string-api-zod/server/feeds/tests/user-feed.test.ts#L291

Reply:
Excellent! I know I saw a bunch of other empty tests with TODO comments when I was troubleshooting, so after this PR I'll probably hop in and do a cleanup where I make all of those `test.todo` as well.